source/auth/auth.rst

```diff
@@ -754,9 +754,11 @@ Client Second
    { 
        "a" : "AWS4-HMAC-SHA256 Credential=AKIAICGVLKOKZVY3X3DA/20191107/us-east-1/sts/aws4_request, SignedHeaders=content-length;content-type;host;x-amz-date;x-mongodb-gs2-cb-flag;x-mongodb-server-nonce, Signature=ab62ce1c75f19c4c8b918b2ed63b46512765ed9b8bb5d79b374ae83eeac11f55",
        "d" : "20191107T002607Z"
+       "t" : "<security_token>"
    }
 |
-Each message above will be encoded as BSON V1.1 objects and sent to the peer as the value of ``payload``. Therefore, the SASL conversation would appear as:
+Note that `X-AMZ-Security-Token` is optional. Each message above will be encoded as BSON V1.1 objects and sent to the
```

Comment:
`X-AMZ-Security-Token` is required when using temporary credentials. When using regular credentials, it MUST be omitted.

Reply:
Done